src/codeu/chat/client/commandline/Chat.java

```diff
@@ -517,7 +628,8 @@ public void invoke(List<String> args) {
         System.out.println("Conversation Info:");
         System.out.format("  Title : %s\n", conversation.conversation.title);
         System.out.format("  Id    : UUID:%s\n", conversation.conversation.id);
-        System.out.format("  Owner : %s\n", conversation.conversation.owner);
+        System.out.format("  Creator  : %s\n", conversation.conversation.owner);
+        System.out.format("  Access Level: %s\n", conversation.getUserAccessLevel().getAccessLevel());
```

Comment:
Nit: I would clarify that this was the "Default Access Level:" rather than the "Access Level:".